CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 * Breaking: Upgrade Jest to 22 ([#109](https://github.com/thymikee/jest-preset-angular/pull/109))
 * Breaking: Upgrade `ts-jest` to 22 ([#109](https://github.com/thymikee/jest-preset-angular/pull/109))
 * Chore: Get rid of explicit `jsdom` dependency and custom test environment
+* Fix: Upgrade example app to Angular 5.2.x using Angular CLI 1.6.6
+* Fix: Add @types/jest as a package dependency
+* Fix: Adjust README for easy installation guide and add configuration section with vendor libaries like jQuery etc..
```

Comment:
Can you move these to master change log and link to PT?

Reply:
do you mean I should commit these changes to CHANGELOG.md on master branch and mention these changes with PR behind ?

Reply:
No, by master I mean master changelog. The first line of this file states:
```
## Changelog (master)
```
Since these "fixes" are not part of `v5.0.0` release, let's just move them up:
```
## Changelog (master)

* Feature: Simplify installation by adding @types/jest as a package dependency ([#116](...))
* Chore: Upgrade example app to Angular 5.2 using Angular CLI 1.6 ([#116](...))

### v5.0.0
**v5.0.0 stuff, do not touch**
```
And later with a followup PR:
```
* Docs: Add a configuration section with vendor libraries like jQuery ([#117](...))
```

Reply:
ah ok I got it